controller/instance_manager_controller.go

```diff
@@ -536,7 +532,7 @@ func (imc *InstanceManagerController) syncLogSettingsToInstanceManagerPod(im *lo
 	defer client.Close()
 
 	settingNames := []types.SettingName{
-		types.SettingNameV2DataEngineLogLevel,
+		types.SettingNameLogLevel,
 		types.SettingNameV2DataEngineLogFlags,
```

Comment:
Rename `SettingNameV2DataEngineLogFlags` to `SettingNameLogFlags`. Make it generalized although v1 data engine doesn't need it.